bitcoin/dogeblock.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020 Coinbase, Inc.
```

Comment:
Please change this to:
```
// Copyright 2021 Dogecoin Rosetta Developers
```

Reply:
We'll need to alter the license check, the license check ensures that Coinbase, Inc is the copyright holder.

Reply:
Okay. Imma disable that check in a separate PR then, because they're only that on the code we reuse, not on the code we write ourselves. On it.